pkg/storage/SQL/plan_document_solution_links_create.sql

```diff
@@ -0,0 +1,25 @@
+INSERT INTO plan_document_solution_link (
```

Comment:
I think we can update this to take an array vs doing this individually for each link